Add Czech

srvlrl opened the pull request:
# Added Czech Language 🇨🇿

- Created cs-cz.json with Czech language translations
- Added cs-cz into languages.json

srvlrl committed:
Create cs-cz.json

Created czech language translation for CollabVM

srvlrl committed:
Update languages.json

Added cs-cz language to the list

srvlrl committed:
Update cs-cz.json

Small gramatical changes

srvlrl committed:
Updated Welcome Modal Body translation to reflect recent changes

Updated kWelcomeModal_Body translation to reflect changes from the main repository

srvlrl committed:
Merge branch 'computernewb:master' into master

srvlrl committed:
Merge branch 'computernewb:master' into master

elijahr2411 reviewed (changes requested):
I apologize @srvlrl, I let this slip by until now. Would you be able to update this to include the localization keys included in the latest webapp updates? Thank you

srvlrl committed:
Merge remote-tracking branch 'upstream/master'

srvlrl committed:
Updated translations

Added new translations due to new labels in the main repository, updated some old translations to make them easier to understand.

srvlrl committed:
Fixed formatting issue

Issued fix based on output from the "Check formatting" action.

srvlrl committed:
Small changes to the czech translation

Small grammatical and format tweaks. Also replaced display name with my GitHub username under "author"

srvlrl committed:
Small wording change

Tweaked a certain translation after finding out where it was used.
